index.js

```diff
@@ -0,0 +1,47 @@
+let library = [];
+
+function getInput() {
+  const book = {};
+  book.title = document.getElementById('bookTitle').value;
+  book.author = document.getElementById('bookAuthor').value;
```

Comment:
- [ ] OPTIONAL: It would be nice if you can empty the input fields after you save the values in a book object. :pray:

Reply:
Thank you for recommend this implementation we will work on it once re-factorize our code.